test/test_db.py

```diff
@@ -1,11 +1,19 @@
+import unittest
+
 from haystack.database.sql import SQLDocumentStore
 from haystack.indexing.io import write_documents_to_db
 
 
-def test_db_write_read():
-    sql_document_store = SQLDocumentStore()
-    write_documents_to_db(document_store=sql_document_store, document_dir="samples/docs")
-    documents = sql_document_store.get_all_documents()
-    assert len(documents) == 2
-    doc = sql_document_store.get_document_by_id("1")
-    assert doc.keys() == {"id", "name", "text", "tags"}
+class TestDB(unittest.TestCase):
```

Comment:
For the test cases, how about using `pytest` instead of unittest? We have been using it in FARM and the experience has been good so far! Especially, the fixtures(with scopes), parameterizing, and benchmarking were useful.

Reply:
Sounds good, I can migrate these to pytest

Reply:
I've made changes in inmemory_store PR, closing this PR